samples/node-headless-ssr-experience-edge/README.md

```diff
@@ -0,0 +1,48 @@
+# node/express scaffolding for SSR using Sitecore Experience Edge
+
+Sitecore JSS for SSR using Experience Edge is considered experimental.
+
+> This is a sample setup that is not officially supported by Sitecore.
+
+This is a sample setup showing one of how you can configure rendering server on top of node.js and Express using Experience Edge. It performs only render of the sample app, quering layout data and dictionary data without proxying requests.
+
+## Pre-requisites
+
+1. Your instance needs to be configured with Headless Services Module and the API Key provisioned.
+
+	### // TODO: document how to test GraphQL queries
+
+1. If necessary, update your app's GraphQL components to be compatible with Experience Edge schema. See examples in [Next.js sample app](../nextjs).
```

Comment:
@illiakovalenko We should probably add a note to this step calling out the need to use the "sc_apikey" header. See related change here:
https://github.com/Sitecore/jss/commit/0e5cbc203badeced4673dd5f0af953b2c8dbf9d8#diff-5f213009f6b7b90a7f23ddfd792ea643c7b1ea84b1413601e994962332e081fcR55

Reply:
Added description about sc_apikey header